src/components/KeeperFooter.tsx

```diff
@@ -39,7 +39,9 @@ export const KeeperFooter = ({
               onPress={item.onPress}
               disabled={item.disabled}
             >
-              <item.Icon />
+              <Box backgroundColor={`${colorMode}.RussetBrown`} style={styles.circle}>
```

Comment:
Make sure this is common in all places where KeeperFooter is being used

Reply:
Checked in the design. It is same.